src/observability/summarizer.go

```diff
@@ -164,6 +185,28 @@ func convertSysLogToSysSummaryMap(syslogs []*pb.Log) {
 			sysSummary.Destination = strings.Split(syslog.Resource, " ")[0]
 		}
 
+		if syslog.Operation == types.OperationTypeSyscall {
+			if existingClustername != syslog.ClusterName {
+				_, services, _, pods, err = cluster.GetAllClusterResources(syslog.ClusterName)
+				if err == nil {
+					existingClustername = syslog.ClusterName
+				}
+			}
+
+			ParentProcess, ChildProcess, Syscall, Parameters, err := extractSyscallInfoFromSystemLog(*syslog, pods, services)
+
+			if err != nil {
+				continue
+			}
+			sysSummary.Source = ParentProcess
```

Comment:
Is it not possible to use Source and Resource from logs directly ?

Reply:
We only need ParentProcessName and ProcessName for syscall and resource is not necessary.
FYI: we have edited the "ParentProcessName -> Source" when we are working "summary data".